tasks/build/download_node_builds.js

```diff
@@ -68,17 +68,75 @@ module.exports = function (grunt) {
     var bytes = parseInt(resp.headers['content-length'], 10) || 'unknown number of';
     var mb = ((bytes / 1024) / 1024).toFixed(2);
     grunt.log.ok(`downloading ${platform.name} - ${mb} mb`);
+
+    await platform.downloadPromise;
+    grunt.log.ok(`${platform.name} download complete`);
+    let file = platform.nodeUrl.substr(platform.nodeUrl.lastIndexOf('/') + 1);
```

Comment:
Rather than use this write this complex expression multiple times, maybe declare this variable at the beginning of the function? Also I wonder if [Path.basename](https://nodejs.org/dist/latest-v4.x/docs/api/path.html#path_path_basename_p_ext) works on urls?